source/UF/Truncations.lagda

```diff
@@ -1,5 +1,7 @@
 Ian Ray, 07/23/2024
```

Comment:
Could we please not write the date in the American system? Either write `7th September 2024` or `2024-07-07`. Because many of the contributors are based in the UK, using any convention with `99/99/99` will be ambiguous. Not in this example, because there is no month 23, of course, but still we should avoid this notation for dates. In all other files you authored as well. Thanks.

Reply:
I fixed this in all files.